app/_components/protocol/_protocol.scss

```diff
@@ -0,0 +1,65 @@
+:target + .app-protocol {
+  outline: $govuk-focus-width solid govuk-colour("light-grey");
```

Comment:
What's this doing? I couldn't see it in action

Reply:
It’s subtle (too subtle)? It adds a border to the summary card when it’s the focus on the URL fragment. Wanted to use yellow focus colour, but it’s not actually focused, so…